source/install-riallto-linux.rst

```diff
@@ -0,0 +1,59 @@
+.. _install-riallto-Linux:
+
+Install Riallto on Linux 
+===========================
+
+Currently there is support for bringing up Riallto on Ubuntu 24.04 with docker.
+To use Riallto on Linux requires the use of the [xdna-driver](https://github.com/amd/xdna-driver) which is installed as part of the setup.
+This driver requires version 6.10+ of the linux kernel, these scripts will upgrade a standard Ubuntu 24.04 installation to this kernel version. __Using this kernel version will require disabling secure boot on your device.__
+
+**Currently there is no support for the section 5 notebooks with the Linux version of Riallto.**
+
+Install Steps
+-------------
+
+On an NPU enabled laptop running Ubuntu 24.04.
+
+1. __Setup Docker.__
+You can follow the steps [here](https://docs.docker.com/engine/install/ubuntu/).
+
+2. __Add your user to the docker user group and then relogin.__ 
+
+```
+sudo usermod -aG docker $USER ; exit
+```
+
+3. __Obtain a license file for Riallto.__
+
+Please follow the guide here :ref:`prerequisites-aie-license`
```

Comment:
This document asks users to get the MAC address using wsl, it should be updated as well.

Reply:
good catch.